src/main/java/net/snowflake/ingest/utils/Constants.java

```diff
@@ -8,14 +8,21 @@
 public class Constants {
 
   // Client level constants
-  public static final String USER_NAME = "snowflake.user.name";
+  public static final String USER = "user";
+  public static final String ACCOUNT = "account";
+  public static final String PORT = "port";
+  public static final String SSL = "ssl";
+  public static final String DATABASE = "database";
+  public static final String SCHEMA = "schema";
+  public static final String CONNECT_STRING = "connect_string";
+  public static final String HOST = "host";
+  public static final String WAREHOUSE = "warehouse";
+  public static final String SCHEME = "scheme";
   public static final String ACCOUNT_URL = "snowflake.url.name";
-  public static final String PRIVATE_KEY = "snowflake.private.key";
-  public static final String ROLE_NAME = "snowflake.role.name";
+  public static final String PRIVATE_KEY = "private_key";
+  public static final String ROLE = "role";
```

Comment:
Could you remove `snowflake.` prefix in ACCOUNT_URL and PRIVATE_KEY_PASSPHRASE as well to make it consistent?